src/nonlin_svm_model.jl

```diff
@@ -0,0 +1,155 @@
+export tanh_train_model, tanh_test_model#, tan_nls_model
+
+function tan_data_train()
```

Comment:
Could we do something like
```julia
function tan_data_train(dataset = MLDatasets.MNIST)
  A, b = dataset.traindata()
  # ... etc.
```
so we can load different datasets?

Reply:
If you want to use dataset in particular, I suppose we could. How would that work with something like my issue now (where I can't download those files)? Could you separate it into cases with "typeof(dataset)==function" if/then - so then it would pick out raw data from a dataset?

Reply:
Well your case is a bit particular because you're behind a firewall of some sort. But I thought you claimed earlier that any other data set from MLDataSets could be used. Is that true?

Reply:
Not sure actually, I thought you guys decided further up in the thread that we should be able to put in any dataset (not MLDatasets in particular). I was just wondering if somebody just wanted to supply A and b instead of using a julia-package-specfic data structure.

Reply:
Ok but by that I meant "download a dataset and use it inside this data-fitting problem". Any dataset from MLDatasets would be fine. But will that work out of the box?